build_tools/github_action/configure_ci_test.py

```diff
@@ -31,7 +31,7 @@ def test_run_ci_if_source_file_and_unrelated_workflow_file_edited(self):
     def test_valid_linux_workflow_dispatch_matrix_generator(self):
         build_families = {"amdgpu_families": "   gfx94X ,|.\\,  gfx1200"}
         linux_target_output = configure_ci.matrix_generator(
-            False, True, False, {}, build_families, "linux"
+            False, True, False, False, {}, build_families, "linux"
         )
```

Comment:
These test cases are getting hard to read with so many boolean arguments. Switch to using keyword arguments instead of positional arguments?

```python
configure_ci.matrix_generator(
  is_pull_request=False,
  is_workflow_dispatch=True,
  ...
```